src/components/Blocks/Pagination.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import classNames from 'classnames';
+import { IconSlideLeft, IconSlideRight } from '../../utils/Icons';
+
+interface Props {
+  currentPage: number;
+  setCurrentPage: React.Dispatch<React.SetStateAction<number>>
+  buttons: number[];
+}
+
+const Pagination: React.FC<Props> = ({
+  currentPage, setCurrentPage, buttons,
+}) => {
+  return (
+    <div className="section-catalog__pagination" data-cy="pagination">
+      <button
+        className="slider-button"
+        type="button"
+        disabled={currentPage === 1}
+        onClick={() => {
+          setCurrentPage(currentPage - 1);
+          window.scrollTo(0, 270);
+        }}
```

Comment:
Move to separate function  (transfer the function to a variable, and here only write the name of the function). Fix all cases